### PR TITLE
Logged in functionality

### PR DESCRIPTION
**Pull request by zStevenn**
I've added the file ./classes/logged_in.php. This function gets all roles available on our website from our local database, puts it into an array and compares the value to the current userrole that is logged in.

**Userrole is detected based on:**
- $_COOKIES/$_SESSION["logged_in"] = true
- $_COOKIES/$_SESSION["userrole"] = userrole
These variables are set after the log-in process.

**Missing:**
- Login-script.php is currently hard-coded for a customer login only. This needs to be changed so userrole variables are given based on the user.
- Login.php and Login-script.php needs a password check as well.
- Navbar and my profile page href must change based on logged in user. Otherwise the user cannot access their respective pages.

I have used test.php to test the functionality, it seems to be working properly.

### CODE REVIEW

File: classes/logged_in.php [hunk 1]
```diff
@@ -12,9 +41,9 @@ public function __construct()
       }
     }
 
-    public function logged_in()
```
Comment (joranvanerk) [inline classes/logged_in.php]: Amazing work, thank you for your contribution!

File: classes/logged_in.php [hunk 2]
```diff
@@ -1,8 +1,37 @@
 <?php
   class logged_in 
   {
+    public $userrole;
+
     public function __construct()
```
Comment (joranvanerk) [inline classes/logged_in.php:17]: The construct function from php is amazing on line six, you created the PHP function and that's really technical and cool.  We love your contribution and hope that you could use this function more frequently! The function is made on a user-level side and that's fine, line 4 on the public userrole makes it perfect!

File: login-script.php [hunk 3]
```diff
@@ -22,8 +22,8 @@
         $_SESSION["email"] = $record["email"];
         $_SESSION["logged_in"] = true;
         $_SESSION["userrole"] = $userrole;
-        $_SESSION["voornaam"] = $data["voornaam"];
-        $_SESSION["achternaam"] = $data["achternaam"];
+        // $_SESSION["voornaam"] = $data["voornaam"];
+        // $_SESSION["achternaam"] = $data["achternaam"];
         setcookie("email", $record["email"], time()+3600);  /* expire in 1 hour */
```
Comment (T-S43) [inline login-script.php:27]: Maybe the cookie timer can be set higher.

File: classes/logged_in.php [hunk 4]
```diff
@@ -47,34 +47,34 @@ public function check_session()
...
+          //echo "<br>You're not logged in.";
           header("Location: ./login");
         }
       } 
       //Case if neither $_COOKIE nor $_SESSION is set.
       else 
       { 
-        echo "You're not logged in.";
+        //echo "You're not logged in.";
         header("Location: ./index");
```
Comment (T-S43) [inline classes/logged_in.php:78]: Maybe sent them to login other than that nice code.

File: classes/logged_in.php [hunk 5]
```diff
@@ -16,7 +33,7 @@ public function check_session()
       {
         if ($_SESSION["logged_in"] === true)
         {
-          $userrole = $_SESSION["userrole"];
+          
         }
       } 
       //Case if $_COOKIE logged in variable is active
```
Comment (T-S43) [inline classes/logged_in.php]: I love the detailed comments keep it up👍

**Review by T-S43 — COMMENTED** [review]
Amazing code other than the cookie timer, I think it looks good.

**Review by joranvanerk — COMMENTED** [review]
The changes you made on like 8 are perfect, you also made some nice classes to be used in the future.

**Comment by joranvanerk** [comment]
Thank you for your contribution, it's an amazing feature, you used the classes and that's perfect.

**Pull request merged**